lib/resources/Disputes.js

```diff
@@ -0,0 +1,21 @@
+'use strict';
+
+var StripeResource = require('../StripeResource');
+var stripeMethod = StripeResource.method;
+
+module.exports = StripeResource.extend({
+
+  path: 'disputes',
+
+  includeBasic: [
+    'list', 'retrieve', 'update', 'setMetadata', 'getMetadata'
```

Comment:
So I've thought more about this, and I think a "setEvidence" method isn't needed, as the following works:

```js
stripe.disputes.update('dp_123', {evidence: {customer_name: 'Bob'}})
```